src/index.html

```diff
@@ -16,17 +19,23 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page__body">
+    <form
+      action="#"
+      class="form"
+    >
+      <input
+        class="form__large-searchbar"
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
+      />
+      <input
+        class="form__small-searchbar"
+        type="text"
+        data-qa="small"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
```

Comment:
Each searchbar should be in own form data-qa attributes big/small should be in form tag. And only attribute data-qa="keypress" to input in big search form